src/GlobalStyles.js

```diff
@@ -1,17 +1,29 @@
 import { createGlobalStyle } from 'styled-components/macro';
-@import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,300&display=swap');
 
 export default createGlobalStyle`
-  *{
+*, ::after, ::before {
     box-sizing: border-box;
-  }  
+    font-family: 'Work Sans', sans-serif;
+}
 
-  html {
-      font-family: 'Work Sans', sans-serif;
-      font-size: 1.2em;
-  }
- 
-  body {
-    margin: 0;
-  }
-`;
\ No newline at end of file
+body {
+    margin:0;
+    font-size: 112.5%;
+    background-color: var(--primary-background);
+    color: var(--primary-text);
+}
+
+ :root {
```

Comment:
Please consider defining your colors in the `:root` above the `body`. Or even make it an extra global style component: `<Colors />` or `<CSSVariables />`